flexget/components/sites/sites/passthepopcorn.py

```diff
@@ -311,7 +198,34 @@ def search(self, task, entry, config):
                 for torrent in movie['Torrents']:
                     e = Entry()
 
-                    e['title'] = torrent['ReleaseName']
+                    # Add the PTP qualities to the title so the quality plugin has a better chance
+                    release_res = torrent['Resolution']
+                    release_res = release_res.replace(
+                        "PAL", "576p"
+                    )  # Common PAL DVD vertial resolution
+                    release_res = release_res.replace(
+                        "NTSC", "480p"
+                    )  # Common NTSC DVD vertial resolution
+                    # many older releases have a resolution defined as 624x480 for example this will split the value at take the hight
+                    tsplit = release_res.split("x", 1)
+                    if len(tsplit) > 1:
+                        release_res = tsplit[1] + 'p'
+
+                    release_name = torrent['ReleaseName'].replace(
+                        "4K", ""
+                    )  # Remove 4K from release name as that is often the source not the torrent resolution it confuses the quality plugin
+                    e['title'] = (
+                        release_name
+                        + ' ['
+                        + release_res
+                        + ' '
+                        + torrent['Source']
+                        + ' '
+                        + torrent['Codec']
+                        + ' '
+                        + torrent['Container']
+                        + ']'
+                    )
```

Comment:
inb4: Use an f-String instead

Reply:
It now uses an f-string. however there are now about a hundred ways you could interpret f-string formatting in python.

Reply:
further googled what an f-string is should be fixed.. not really a regular python dev so any bad habits are being copy and pasted from other existing code that has formatted strings everywhere. didn't get the distinction or benefit of the f string without further reading.